src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/base.xml"/>
+
+    <appender name="ALERT_AUDIT" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <encoder>
+			<pattern>${FILE_LOG_PATTERN}</pattern>
+		</encoder>
+		<file>log/alertAudit.log</file>
+		<rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+			<cleanHistoryOnStart>${LOG_FILE_CLEAN_HISTORY_ON_START:-false}</cleanHistoryOnStart>
+			<fileNamePattern>${ROLLING_FILE_NAME_PATTERN:-log/alertAudit.log.%d{yyyy-MM-dd}.%i.gz}</fileNamePattern>
+			<maxFileSize>${LOG_FILE_MAX_SIZE:-10MB}</maxFileSize>
```

Comment:
This seems a little small. What made you choose this default value? I realized that it's a rolling log file, but it seems like this size could produce a lot of files. I would expect somewhere between 50MB and 250MB.

Reply:
The appender here is actually a copy of the default file appender used by spring to generate our current blackduck-alert.log file. The only difference here is the path that it is being logged to. The default values here are already the same ones we are using in the alert logs.

Reply:
I agree with Gavin here, the default for both should be increased to 50 - 250MB.

Reply:
This value can also be controlled for both logs by setting logging.file.max-size to the desired size, might be easier to set it there depending on the customer and how big the logs they produce are

Reply:
The property logging.file.max-size would be a property in the application.properties file which isn't exposed to the customer.  I would suggest adding that property to the application.properties file and also creating an environment variable so it can be controlled by support, devops, or the customer.
Something like this.
`logging.file.max-size=${ALERT_LOG_FILE_SIZE:50MB}`